grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServerInterceptorTest.java

```diff
@@ -46,11 +45,10 @@
     @RegisterExtension
     static ServerExtension server = new ServerExtension() {
         @Override
-        protected void configure(ServerBuilder sb) throws Exception {
+        protected void configure(ServerBuilder sb) {
             sb.service(GrpcService.builder()
-                                  .addService(ServerInterceptors.intercept(
-                                          new TestServiceImpl(Executors.newSingleThreadScheduledExecutor()),
-                                          NoPassInterceptor.INSTANCE))
+                                  .addService(new TestServiceImpl(Executors.newSingleThreadScheduledExecutor()))
+                                  .intercept(NoPassInterceptor.INSTANCE)
```

Comment:
Could you add a test that verifies the injected interceptors correctly works such as setting a value to `Metadata` and accessing it from a service?

Reply:
> Could you add a test that verifies the injected interceptors correctly works such as setting a value to `Metadata` and accessing it from a service?

Doesn't the injected `NoPassInterceptor.INSTANCE` (and its response exception status) verify they work correctly?

Reply:
Ah got it. 😆